### PR TITLE
Nsf connie

### PR DESCRIPTION
**Pull request by bryanwweber**
* Add Connie Gao to the Steering Committee
* Add a blog post about the NSF funding

### CODE REVIEW

File: posts/nsf-awards-cantera-project-25m-through-2022.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- tags: funding
+- category: null
+- link: null
+- description: null
+- type: text
+---
+
+# NSF awards Cantera Project $2.5M through 2022
+
+We are proud to announce that NSF has awarded funds to members of the development team and steering committee, under the _Cyberinfrastructure for Sustained Scientific Innovation (CSSI)_ program. The proposal, titled _Extensible and community-driven thermodynamics, transport, and chemical kinetics modeling with Cantera: expanding to diverse scientific domains_, was awarded $2.5M USD to enable new scientific modeling capabilities, improvements to the Cantera software architecture, and community development activities. These activities will help further Cantera's primary objective, which is a generalizable software package to assist users in a variety of fields with a range of calculations related to thermodynamics, chemical kinetics, and species transport.
```
Comment (kyleniemeyer) [inline posts/nsf-awards-cantera-project-25m-through-2022.md]: perhaps link to the CSSI program? https://www.nsf.gov/pubs/2019/nsf19548/nsf19548.htm

Comment (kyleniemeyer) [inline posts/nsf-awards-cantera-project-25m-through-2022.md]: should we make the proposal public? The Jupyter project also just got a big NSF grant, and put their proposal on Zenodo: https://blog.jupyter.org/jupyter-meets-the-earth-1b0eb33c83f

Comment (kyleniemeyer) [inline posts/nsf-awards-cantera-project-25m-through-2022.md]: language edits:
> to enable new scientific modeling capabilities, improvements to the Cantera software architecture, and community development activities

to

> to enable new scientific modeling capabilities, make improvements to the Cantera software architecture, and perform community development activities.

Comment (bryanwweber) [reply posts/nsf-awards-cantera-project-25m-through-2022.md]: We should discuss this in the group. We can update the blog post (or make a new one) if we decide to release it.

**Pull request merged**